R/lazyframe-frame.R

```diff
@@ -45,7 +45,6 @@ wrap.PlRLazyFrame <- function(x, ...) {
   self
 }
 
-# TODO: link to pl__select
```

Comment:
Why?

Reply:
I don't see how this should be mentioned in the docs of `$select()` but I can add something if you have a suggestion

Reply:
My point is why are you deleting TODOs for me without my consent. Please do not delete them.